.github/workflows/storybookBuild.yml

```diff
@@ -2,6 +2,8 @@ name: Storybook Release
 
 on:
   pull_request:
+    paths:
+      - 'webapp_frontend/src/**'
```

Comment:
@open-schnick why does this not work, it is running everytime

Reply:
because it uses the master yml.

Reply:
its kinda stupid. now it should work. but when you are updating a workflow it still uses the master one, but when creating one its used although its not on master.